app/src/main/java/com/hifeful/mealmania/domain/model/Meal.kt

```diff
@@ -6,6 +6,18 @@ import androidx.room.Entity
 import androidx.room.PrimaryKey
 import kotlinx.parcelize.Parcelize
 
+/**
+ *  1️⃣ It should be inside the data layer as it's a database entity(i.e., MealEntity)
+ *  2️⃣ In the domain layer you are going to have a model (i.e., Meal) which is mapped
+ *     in the data layer based on received entity
+ *  3️⃣ Domain has only models and repository contracts
+ *     In some cases, use-cases as well
+ *
+ *  🎁 When we have multi-modular architecture:
+ *     > the domain layer shouldn't have any dependencies, even to another domain
+ *     > the data layer should have only the dedicated domain dependency, and no another data layers
+ *     > of course, there might be exception, but usually we should avoid it
+ */
```

Comment:
Agree. I knew that but decided to not duplicate classes for such a simple application. Of course in the multi-modular architecture, it's mandatory.